gulpfile.js

```diff
@@ -129,7 +131,21 @@ defineBuildTask('HTMLImports');
 defineBuildTask('ShadowDOM');
 defineBuildTask('MutationObserver');
 
-gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements', 
+gulp.task('CustomElementsV1', function () {
+  return gulp.src('./src/CustomElements/v1/CustomElements.js', {base: './'})
+      .pipe(closureCompiler({
```

Comment:
@trusktr This library neither imports or exports anything in the module sense. Because it's a polyfill, it's directly patching prototypes and defining a global. I'm not sure modules would buy us anything, while they would still require compilation.

I'm not sure why anyone would need to define their own workflow. We will distribute the built files.